client/src/components/publications/Publications.js

```diff
@@ -98,6 +99,15 @@ const Publications = () => {
   useEffect(() => {
     const sortedPublication = sortPublications(teamPublications, options.sortBy);
     setPublications(sortedPublication);
+    const newlyAddedPublications = teamPublications.filter((pub) => pub.newlyAdded === true);
```

Comment:
do we need to `=== true`? or can we omit it and just have `(pub) => pub.newlyAdded`? Also, this is a separate issue, but newlyAdded could be renamed to `isNewlyAdded` to make it more clear what we're referring to.